bioptim/examples/getting_started/pendulum.py

```diff
@@ -29,7 +29,8 @@ def prepare_ocp(
     biorbd_model_path: str,
     final_time: float,
     n_shooting: int,
-    ode_solver: OdeSolver = OdeSolver.RK4(),
+    ode_solver: OdeSolver = OdeSolver.COLLOCATION(polynomial_degree=3, method="radau"),
+    # ode_solver: OdeSolver = OdeSolver.RK4(),
```

Comment:
Do not push changes to the examples, create a new one if this is what you want to showcase

Reply:
Sorry.